charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -365,60 +372,62 @@ spec:
         resources:
 {{ toYaml $operatorValues.spec.authProxy.resources | indent 10 }}
         volumeMounts:
+{{- if $operatorValues.spec.config.tls.api.enabled }}
         - mountPath: /etc/tls
           name: api-tls
+{{- end }}
         - mountPath: /etc/proxy/cookie-secret
           name: cookie-secret
         - mountPath: /etc/proxy/htpasswd
           name: htpasswd-secret
-{{- if $operatorValues.spec.authProxy.authenticatedEmailsEnabled  }}
+{{- if $operatorValues.spec.authProxy.authenticatedEmails.enabled  }}
         - mountPath: /etc/proxy/authenticated-emails
           name: authenticated-emails-secret
 {{- end }}
-{{- end }}
+{{- end }}{{/* end of authProxy.enabled */}}
```

Comment:
nice, I had a hard time following this when I was initially looking at the reporting operator deployment.